Eternal Review

kinbiko opened the pull request.

kinbiko commented on Brewfile:
```diff
@@ -0,0 +1,31 @@
+tap "caskroom/cask"
```
This file is outdated, and when I tried to install using this file on another computer it failed.
I think it's a good idea to keep these, but I would like to have a way of confirming that it does work.

kinbiko commented on chunkwmrc:
```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```
I really like this project.
But:
- There are too many bugs for me to use this consistently (In particular when switching between the number of monitors
- A lot of new shortcuts to learn
- I don't know enough C++ to contribute
- Has external dependency to bind keys.

I should probably scrap this and revisit in the future

kinbiko commented on khdrc:
```diff
@@ -0,0 +1,136 @@
+# !!!
```
C.f. comment about chunkwm.

kinbiko commented on tmux.conf:
```diff
@@ -0,0 +1,1034 @@
+# cat << ►_◄ > /dev/null
```
Long term project:
Build my own tmux.conf and tmux.conf.local. and understand what is going on.
Right now it's copy pasta with patches.

kinbiko commented on vim/ftplugin/coffee.vim:
```diff
@@ -0,0 +1,18 @@
+"Run mocha test
+nnoremap <localleader>1 :w<CR>yi':! clear;npm run testlog -- -g "<C-r>0"<CR>
```
Explore doing this with vimux, c.f. ruby config

kinbiko commented on vim/install-plugins.vim:
```diff
@@ -0,0 +1,61 @@
+Plug 'Xuyuanp/nerdtree-git-plugin' "Nerdtree + git highlighting
+Plug 'wincent/terminus' "Cursor changes in different modes, mouse support
+Plug '/usr/local/opt/fzf' | Plug 'junegunn/fzf.vim' "fzf in vim
+Plug 'rking/ag.vim' "search inside files
```
See if there's any way of getting incremental results for this. If there is that would be awesome!

kinbiko commented on vim/mapping-config.vim:
```diff
@@ -0,0 +1,122 @@
+noremap <leader>. :b#<CR>
+
+"Follow and return from ctagged files
+nnoremap ] g<C-]>
```
This is `Cmd+]` in intelliJ, can probably be mapped to an action for consistency

kinbiko commented on vim/mapping-config.vim:
```diff
@@ -0,0 +1,122 @@
+nnoremap [ <C-t>
+
+"Fuzzy-find any file in this git repo with ?
+nnoremap ? :GFiles<CR>
```
Would be ideal if this didn't depend on being inside a repo. I.e. being able to hit ? and find any file within the current working directory, as long as it's not gitignored.

kinbiko commented on vim/mapping-config.vim:
```diff
@@ -0,0 +1,122 @@
+sunmap e
+
+"Make H be home, L be end
+noremap H ^
```
I don't really use H and L for this. Could be valuable key-real estate.

kinbiko commented on vim/mapping-config.vim:
```diff
@@ -0,0 +1,122 @@
+
+"Literally the best keybindings I know
+nmap <BS> {
+nnoremap <expr> <CR> empty(&buftype) ? '}' : '<CR>'
```
Mmmm... This works really well, apart from the help window.

kinbiko commented on vim/mapping-config.vim:
```diff
@@ -0,0 +1,122 @@
+map * <Plug>(asterisk-z#)
+
+"Because :w is too annoying
+nnoremap s :w<CR>
```
Change `:w` and `:wq` to echo "Use 's'" and "Use ZZ"

kinbiko commented on vim/mapping-config.vim:
```diff
@@ -0,0 +1,122 @@
+"}}}
+
+"Make a vertical split with vv, and go to tag if on one
+nnoremap <silent> vv :vsp <CR>:exec("tag ".expand("<cword>"))<CR>
```
This could be a lot better:
Do a split opening the given tag if present, otherwise try and `gf` the file, otherwise open the same file in a split, without any error messages.

kinbiko commented on vim/nerdtree-config.vim:
```diff
@@ -0,0 +1,23 @@
+  exec 'autocmd filetype nerdtree syn match ' . a:extension .' #^\s\+.*'. a:extension .'$#'
+endfunction
+
+call NERDTreeHighlightFile('md', 'blue', 'none', '#3366FF', '#151515')
```
We can do a lot more than that!!

kinbiko commented on vim/spell/en.utf-8.add:
```diff
@@ -0,0 +1,20 @@
+v4
+SuperFramework
+gorram
+dp
```
This ain't a word

kinbiko commented on vim/spell/en.utf-8.add:
```diff
@@ -0,0 +1,20 @@
+dp
+kinbiko
+Guldbrandsen
+th
```
This ain't a word

kinbiko commented on vim/spell/en.utf-8.add:
```diff
@@ -0,0 +1,20 @@
+Microservices
```
Is there a large programmer's dictionary I can use as a starting point?

kinbiko commented on vim/vim:
```diff
@@ -0,0 +1 @@
+repos/dotfiles/vim
```
Remove cyclic symlink

kinbiko commented on vimrc:
```diff
@@ -0,0 +1,54 @@
+source ~/.vim/airline-config.vim
+source ~/.vim/syntastic-config.vim
+source ~/.vim/plugin-config.vim
+source ~/.vim/nerdtree-config.vim
```
A few of these sources should really be part of the `plugin-config.vim` file instead.

kinbiko commented on vimrc:
```diff
@@ -0,0 +1,54 @@
+let g:syntastic_style_warning_symbol = '≈'
+
+" max line lengh that prettier will wrap on
+let g:prettier#config#print_width = 80
```
Extract prettier config to `plugin-config.vim`

kinbiko commented on vimrc:
```diff
@@ -0,0 +1,54 @@
+let g:prettier#autoformat = 0
+autocmd BufWritePre *.js,*.jsx,*.mjs,*.ts,*.tsx,*.css,*.less,*.scss,*.json,*.graphql PrettierAsync
+
+nnoremap <c-N> :GitGutterNextHunk<cr>
```
Move to `mapping-config.vim`.

kinbiko commented on vimrc:
```diff
@@ -0,0 +1,54 @@
+" when running at every change you may want to disable quickfix
+let g:prettier#quickfix_enabled = 0
+let g:prettier#autoformat = 0
+autocmd BufWritePre *.js,*.jsx,*.mjs,*.ts,*.tsx,*.css,*.less,*.scss,*.json,*.graphql PrettierAsync
```
Check if .md is supported in the vim plugin yet.

kinbiko commented on zsh/extract.sh:
```diff
@@ -0,0 +1,20 @@
+extract () {
```
Add hat tip to devrant user who came up with this

kinbiko commented on zsh/tmuxinator.zsh:
```diff
@@ -0,0 +1,29 @@
+_tmuxinator() {
```
Remove

kinbiko commented on zshrc:
```diff
@@ -0,0 +1,107 @@
+alias gc="git commit"
+alias gdc="git diff --cached"
+alias ga="git add ."
+alias gap="git add -p"
```
duplicate this with `add`

kinbiko commented on zshrc:
```diff
@@ -0,0 +1,107 @@
+alias ga="git add ."
+alias gap="git add -p"
+alias commit="git commit"
+alias pull="git pull"
```
Make this pull also do a prune

wsdjeg commented:
I like this idea, It is easy for people to review your config and add line comment.

kinbiko committed:
[vim] Lower go test timeout

kinbiko committed:
[vim] Make nvim compatible

kinbiko committed:
[vim] Use neovim by default

- Also sets up denite with some example config

kinbiko committed:
[vim] Fuzzy-find files including unstaged with ?

kinbiko committed:
[vim] Add a completion engine that might work

kinbiko committed:
[vim] fix folds in vimrc

kinbiko committed:
[vim] Improve go autocompletion

kinbiko committed:
[ctags] Delete langdefs I don't use

kinbiko committed:
[vim] ftplugin spring cleaning

kinbiko committed:
[vim] Add some new words to dictionary

kinbiko committed:
[vim] Add sideways for parameter swapping

kinbiko committed:
[vim] Record list of safe, future key mappings

kinbiko committed:
[zsh] Clear for certain git commands

kinbiko committed:
[tmux] Keep path when creating windows

kinbiko committed:
[zsh] Don't update brew every time I install

kinbiko committed:
[zsh] This is basically unix blasphemy

kinbiko committed:
[vim] Associate .feature with ruby

kinbiko committed:
[vim] Autocomplete in the correct direction

kinbiko committed:
[vim] Always use neovim

kinbiko committed:
[vim] API is a word

kinbiko committed:
[tmux] Add fpp support

kinbiko committed:
[ergodox] Add ergodox config

kinbiko committed:
[zsh] Show authors of signed commits

kinbiko committed:
la now works using 'vanilla' ls

kinbiko committed:
Transpose lines in visual mode with c-j/k

kinbiko committed:
[alfred] Add alfred settings

kinbiko committed:
[vim] copy+paste within brackets with gb gp

kinbiko committed:
[alfred] sync with other computer

kinbiko committed:
[alfred] Add urban dictionary search

kinbiko committed:
[alfred] todoist cleanup

kinbiko committed:
[alfred] Enable clipboard

kinbiko committed:
[alfred] Dash integration updates

kinbiko committed:
[alfred] Update dash

kinbiko committed:
[vim] Add pretty icons

This used to make my setup *really* slow, but it appears to have no
performance impact anymore after moving over to neovim

kinbiko committed:
[vim] denite improvements

kinbiko committed:
[vim] Use denite to open vimrc

kinbiko committed:
[vim] Make comments red

kinbiko committed:
[vim] javadoc is a word

kinbiko committed:
[vim] habit enforcer to stop doing £

kinbiko committed:
[vim] broken j button means drastic measures

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[vim] Make moving around in visual mode faster

kinbiko committed:
[vim] copy to clipboard with Y

kinbiko committed:
[zsh] Fix scripts path

kinbiko committed:
[vim] Make moving around in visual mode faster

kinbiko committed:
[vim] remove references to vimux

kinbiko committed:
[vim] Update ruby bindings to use leader

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[vim] search results appear in the middle of the window

kinbiko committed:
[alfred] Remove alfred configs from dotfiles

kinbiko committed:
[vim] Add expand-region plugin

kinbiko committed:
[docs] Update README to be more current

kinbiko committed:
[vim] Remove unused fzf plugin

kinbiko committed:
[vifm] Open one pane on start

kinbiko committed:
add glint alias

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
add Japan wallpapers

kinbiko committed:
[zsh] Improve 'go test' alias

kinbiko committed:
[zsh] ctrl+x ctrl+x to open vim for current command

kinbiko committed:
[zsh] Steal a bunch of goodies from wincent

- Don't correct go commands by default
- Make zsh autocomplete more powerful
  - case sensitivity if applicable
  - abbreviate filenames m_t.g => main_test.go
  - substring completion rc -> vimrc
- Show colours in autocomplete
- Append a slash when autocompleting directories

kinbiko committed:
[vim] write JS fat arrows faster with >>

kinbiko committed:
remove vim-expand-region as I'm not using it

kinbiko committed:
[script] Add locdoc

kinbiko committed:
add scripts to path

kinbiko committed:
[vim] Play it a bit more safe with vim-go

kinbiko committed:
[vim] Scrap denite in favour of fzf again

kinbiko committed:
[vim] Easy lookup of definitions with leader-f and ?

kinbiko committed:
[vim] fix lookup by file name

kinbiko committed:
[vim] Spring cleaning

Nevermind the fact that it's the middle of June

kinbiko committed:
[zsh] Add a bash function to get GitHub READMEs

kinbiko committed:
[vim] Make ? work outside repos

kinbiko committed:
[newsboat] Add initial newsboat config

kinbiko committed:
[newsboat] Set up some RSS feeds

kinbiko committed:
[vim] set up coc

kinbiko committed:
[vim] try snippets

Commit:
Start documenting mac install instructions

kinbiko committed:
[zsh] Told you I would do it

kinbiko committed:
[alacritty] add alacritty config

kinbiko committed:
[zsh] Be quieter when running go tests

kinbiko committed:
[zsh] Split gt into two

kinbiko committed:
[vim] Preview substitutions

kinbiko committed:
[ergodox] Revisit ergodox layout and note down their urls

kinbiko committed:
[vim] go tweaks

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[ergodox] Some notes on the latest version

kinbiko committed:
[ergodox] add version 6

kinbiko committed:
[vim] Make vim + go quieter

kinbiko committed:
Remove unused plugins

kinbiko committed:
[zsh] Be a bit more lenient and consistent with gt

kinbiko committed:
[vim] Improve go snippets

kinbiko committed:
[ergodox] add notes on current layout

kinbiko committed:
[alacritty] don't close on cmd+w

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[ergodox] New layout

kinbiko committed:
[vim] Open tags for current word with T

kinbiko committed:
[vim] find-in-nerdtree and buffer bindings

kinbiko committed:
[ergodox] Suggestions for next version

kinbiko committed:
[vim] go subtest snippet

kinbiko committed:
[vim] Add more Go snippets

kinbiko committed:
[ergodox] Version 8

kinbiko committed:
[vim] add ObjectID snippet

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[vim] I don't use these bindings

kinbiko committed:
[vim] Automatically regenerate tags for me

kinbiko committed:
[vim] Get a preview with B and ?

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[vim] Make comments yellow

This makes comments stand out more than code (personal preference) but
still preserves a reasonable colour in test coverage.

kinbiko committed:
[ergodox] Suggestions for next version

kinbiko committed:
[vim] Ag uses preview

kinbiko committed:
[zsh] Make gt and gtv more resiliant and accurate

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[vim] fix annoying checkbox autocomplete

kinbiko committed:
[zsh] remove vifm aliases

kinbiko committed:
[vim] add shortcuts for Q&A in markdown

kinbiko committed:
[zsh] Remove unused aliases

kinbiko committed:
[vim] Add quick open notes binding

kinbiko committed:
[vim] Activate prettier for markdown

kinbiko committed:
[vim] Exclude mocks and test in tag scans

kinbiko committed:
[vim] Coc improvements

kinbiko committed:
[vim] Add coc-settings

kinbiko committed:
[alacritty] Update config to latest version

kinbiko committed:
[vim] Enable prettier in markdown

kinbiko committed:
[vim] Add words to dictionary

kinbiko committed:
[zsh] Enforce utf-8 when attaching to a session

kinbiko committed:
[ergodox] Add new version idea

kinbiko committed:
[vim] update plug

kinbiko committed:
[vim] handle breaking new version of FzfPreview

kinbiko committed:
[vim] JavaScript spacing rules that match prettier

kinbiko committed:
[ctags] Don't ddos ctags with node_modules

kinbiko committed:
[ergodox] Suggestion for next version

kinbiko committed:
[vim] Missing doc on a binding

kinbiko committed:
[vim] Animate window resizing

kinbiko committed:
[vim] Cleanup of unused Coc bindings

kinbiko committed:
[vim] Use arrow keys to move farther than hjkl

kinbiko committed:
[ergodox] Finally a new keyboard layout

kinbiko committed:
[vim] Stop using BS/CR for {}

kinbiko committed:
[vim] Hide tag flies in NERDTree

kinbiko committed:
[vim] Add a mapping for taking quick notes

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[vim] add small in-buffer navigation mappings

kinbiko committed:
[ergodox] Note for next version

kinbiko committed:
[vim] Spring cleaning

kinbiko committed:
[vim] Move coc-config to dotfiles

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[tmux] Update nord theme

kinbiko committed:
[tmux] Remove broken configs following upgrade

kinbiko committed:
[zsh] Add ignore-whitespace git aliases

kinbiko committed:
[vim] Highlight yanked text

kinbiko committed:
[tmux] Update nord theme version

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[ideavim] Make using Goland a bit more bearable

kinbiko committed:
[vim] Add quickfix plugin

kinbiko committed:
[vim] Hide .idea folder in NERDTree

kinbiko committed:
[vim] Remove misnamed coc config

kinbiko committed:
[vscode] vimrc for vscode vim plugin

kinbiko committed:
[ergodox] Add suggestion for next version

kinbiko committed:
[vim] Kubernetes is a word

kinbiko committed:
[vim] Remove left/right bindings

kinbiko committed:
[vim] Cleanup of after/ftplugin files

- I don't really these filetypes anymore:
  - CoffeScript
  - XML
  - Gherkin
- [go] Simplify ugly test snippet.
- [go] Remove snippet for Mongo ObjectIDs.
- [go] Remove workarounds harking back to when I had to use testify.
- [javascript] Remove snippets that I don't use anymore.
- [markdown] Document what the 'qqq' and 'aaa' abbreviations are for.
- [markdown] Enable Emmet, because why not?

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[tmux] Update nord theme

kinbiko committed:
[other] Remove stuff I don't use anymore

kinbiko committed:
[docs] Update README to more accurately reflect current setup

kinbiko committed:
[ergodox] Clarify next release wishes

kinbiko committed:
[vim] Delete certain settings that I don't use

kinbiko committed:
[vim] Tidy up

kinbiko committed:
[vim] Cleanup fzf and coc bindings

kinbiko committed:
[ergodox] Some thoughts on a major revamp

kinbiko committed:
[vim] Add mapping to toggle coverage in Go

kinbiko committed:
[wallpaper] Pokermon

kinbiko committed:
[zsh] Update 'ga' alias

kinbiko committed:
[vim] TypeScript support

kinbiko committed:
[zsh] Tidy up 'glog' alias and improve performance

kinbiko committed:
[docs] Update README following install on new mac

kinbiko committed:
[alacritty] Migrate to latest config format

kinbiko committed:
[tmux] Delete unused plugins and tidy submodule structure

kinbiko committed:
[vim] Make sure that copying Japanese from vim works fine

kinbiko committed:
[zsh] Autocompletions following brew's instructions

kinbiko committed:
[zsh] Add GPG env var needed for signing via TUI

kinbiko committed:
[docs] Update setup instructions following fresh Mac install notes

kinbiko committed:
[docs] Fix typo in README

kinbiko committed:
[docs] Also install jq

kinbiko committed:
Merge branch 'master' of github.com:kinbiko/dotfiles

kinbiko committed:
[docs] Add clipboard history in Alfred setup

kinbiko committed:
[tmux] Make pane indexes 0 based

kinbiko committed:
[tmux] Disable activity indicators

These mess up the theme anyway

kinbiko committed:
[tmux] Remove references to plugins I haven't used in ages

kinbiko committed:
[tmux] Increase max session name length

kinbiko committed:
[docs] Mention dev tools for go in the README

kinbiko committed:
[zsh] Clean up unused stuff

kinbiko committed:
[tmux] Prefix+f to pass tmux pane to fpp

kinbiko committed:
[docs] Automatically hide menu bar

kinbiko committed:
[vim] Streamline go development

Steal some vim-go settings from fatih's config.

kinbiko committed:
[alacritty] Better support for Linux

kinbiko committed:
[wallpaper] More wallpapers

kinbiko committed:
[vim] Ensure the latest fzf is installed

kinbiko committed:
Merge branch 'master' of ssh://github.com/kinbiko/dotfiles

kinbiko committed:
[vim] Fix typos and lies in plugin comments

kinbiko committed:
[vim] Set up basic rust plugin

kinbiko committed:
[git] Add a global gitignore file

kinbiko committed:
[zsh] Use zenith instead of top

kinbiko committed:
[zsh] Make 'ag' use a pager

kinbiko committed:
[vim] Add a rust println! debug snippet

kinbiko committed:
[zsh] Add 'checkout' shorthand for git + fzf

kinbiko committed:
[vim] Quickly copy link to lines in GitHub

kinbiko committed:
[fzf] Pass via ag to adhere to gitignores

kinbiko committed:
[alacritty] Use buttonless window mode

Granted, this isn't great for Linux, but I don't mind hard copying and
overwriting this value when I'm on a Linux box.

kinbiko committed:
[wallpaper] mantra

kinbiko committed:
[vim] These are words

kinbiko committed:
[docs] Add kokodoko dependency

kinbiko committed:
[vim] Add markdown rendering plugin

kinbiko committed:
[vim] PagerDuty, sadly, is a thing

kinbiko committed:
[wallpaper] Move mantra to correct dir

kinbiko committed:
[ergodox] Gaming layer

kinbiko committed:
[vim] backend is a word

kinbiko committed:
[wallpaper] Add octocat

kinbiko committed:
[tmux] Remove hostname from statusline

kinbiko committed:
[vim] Update vim-plug

kinbiko committed:
[docs] Update README following new mac install

kinbiko committed:
[docs] Mention how to hide fluff dirs

kinbiko committed:
[vim] Add terraform plugin

kinbiko committed:
[vim] Remove unused markdown preview plugin

kinbiko committed:
[vim] Add better TypeScript support

kinbiko committed:
[tmux] Enable true-colours within tmux

kinbiko committed:
[vim] Better support for JS/TS/JSX/TSX

kinbiko committed:
[vim] Start using coc like it should be used

kinbiko committed:
[all-repos] Add initial config

kinbiko committed:
[git] Don't ever commit .idea directory contents

kinbiko committed:
[vim] These are words

kinbiko committed:
[wallpaper] Add Final Fantasy and Kyoto wallpapers

kinbiko committed:
[zsh] Change docker compose alias to subcommand

It used to be a standalone python binary called 'docker-compose' but now
it's built into the docker Go binary, apparently.
It silences a warning on running these aliases.

kinbiko committed:
[ergodox] Add no-inertia mouse idea

kinbiko committed:
[zsh] Clean up zshrc

- Extract an aliases.sh file
- remove unused code
- Add dependency on lazygit

kinbiko committed:
[vim] Rewrite config in Lua

- Rewrite as much of my VimL config in Lua.
  - Any VimL that still exists is evaluated as a string from Lua via
    vim.cmd().
- Start utilizing XDG_CONFIG_HOME environment variable, i.e. put configs
  under ~/.config/ in general on UNIX systems.
  - Ignore most things in this directory apart from nvim/ and
    alacritty/
- Change vim plugin package manager from vim-plug to packer, which has a
  native native Lua API.
- Change the NERDTree origin from 'scrooloose' to 'perservim', as the
  ownership seems to have transferred.
- Spread configuration across multiple files by how they tweak the
  config.
- Make cross-functional (e.g. settings + mapping) explicit, so I don't
  end up with half-removed plugins in the future by forgetting to remove
  a mapping if I remove a setting and a plugin.
- Add a fancy start-screen plugin
- Be explicit about which LSP servers I configure
  - LSPs themselves can be easily installed based on filetype via
    :LSPInstall <filetype>, but are not installed automatically.
- Remove CoC in favour of the built-in LSP client.
- Add Coq for snippets and reading docs easily.
- Add :Trouble plugin for quickly accessing the quickfix window with LSP
  diagnostics.
- Update and move the spelling file, and remove redundant automatic
  updating of spell file VimL snippet.
- Add tokyo night theme.

kinbiko committed:
[alacritty] Move config under XDG_CONFIG_HOME with tokyonight theme

kinbiko committed:
[tmux] Add tokyo-night theme

kinbiko committed:
[gitignore] Add config directory with alacritty and nvim

kinbiko committed:
[docs] Remove redundant instructions in setup

kinbiko committed:
[ctags] Remove ctags as LSPs are a thing now

kinbiko committed:
[zsh] Set an XDG_CONFIG_HOME envvar

kinbiko committed:
[tmux] Move configs under XDG_CONFIG_HOME

kinbiko committed:
[docs] Update tmux symlink

kinbiko committed:
[git] Move global gitignore under XDG_CONFIG_HOME

kinbiko committed:
[git] Add placeholder home-folder gitignore for fresh installs

kinbiko committed:
[docs] Update README now that its simpler

kinbiko committed:
[vim] Give up on an impossible TODO

kinbiko committed:
[vim] Go on a nvim plugin shopping spree

kinbiko committed:
[vim] These are words

kinbiko committed:
[vim] Add numb
